TriforkSwiftExtensions/TriforkSwiftExtensions/String/StringProtocolExtension.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  StringProtocolExtension.swift
+//  TriforkSwiftExtensions
+//
+//  Created by Kim de Vos on 14/02/2019.
+//  Copyright © 2019 Trifork A/S. All rights reserved.
+//
+
+import Foundation
+
+public extension StringProtocol where Self.Index == String.Index {
+    /// Returns a trimmmed string
+    public var trim: String {
```

Comment:
I think this should be a function and should probably be called `trimmed()` instead.  The documentation should also describe that it returns a new string.

Reply:
On second thought, I am not sure whether `trimmed()` or `trimming()` is right? Apple's naming `.append()` vs. `.appending()` should be used as reference 😜

Reply:
I think `.trimmed()` would be best. It tells that it returns something, while `.trimming()` is more mutating?